package.json

```diff
@@ -39,6 +39,9 @@
     "ucompiler-plugin-newline": "^3.0.0"
   },
   "eslintConfig": {
-    "extends": "steelbrain"
+    "extends": "steelbrain",
+    "rules": {
+      "no-duplicate-imports": 0
```

Comment:
This is required to have separate
`import x from 'atom'` and
`import type x from 'atom'`